framework/include/dirackernels/ReporterPointSource.h

```diff
@@ -37,6 +37,10 @@ class ReporterPointSource : public DiracKernel, public ReporterInterface
   const std::vector<Real> & _y_coord;
   ///z coordinate
   const std::vector<Real> & _z_coord;
+  const std::vector<Point> & _point;
   /// map to associate points with their index into the vpp value
   std::map<Point, size_t> _point_to_index;
+
+  const std::vector<Real> _empty_vec = {};
+  const std::vector<Point> _empty_points = {};
```

Comment:
doxygen

Reply:
this code is from @loganharbour and @maxnezdyur 's commits. I dont really want to modify it so it doesnt create conflicts with their PR (which will get in one day?)

Reply:
I don't think yours is going to take that long to get in...

Reply:
removed them